playground/worker/classic-worker.js

```diff
@@ -1,5 +1,5 @@
 let base = `/${self.location.pathname.split('/')[1]}`
-if (base === `/worker-entries`) base = '' // relative base
+if (base.endsWith('.js') || base === `/worker-entries`) base = '' // for dev
```

Comment:
Started getting a lot of warnings in dev
```js
Error: Failed to load url /classic-shared-worker.js/classic.js (resolved id: /classic-shared-worker.js/classic.js). Does the file exist?
 ❯ loadAndTransform packages/vite/src/node/server/transformRequest.ts:242:22
 ❯ processTicksAndRejections node:internal/process/task_queues:95:5
```
I don't know if this is the correct fix, seems the PR is exposing this issue?

Reply:
Seems to happen on `main` too. This fix works for me.